### PR TITLE
add IBC details for quicksilver-2

### PR DESCRIPTION
**Pull request by ajansari95**
add ibc paths for quicksilver and followwing chains :

1.  cosmoshub
2. juno
3. regen
4. stargaze
5. secret
6. umee

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Due to enforcement of alphabetical order, for pair with cosmoshub and juno, quicksilver should be chain2, not chain 1. Please correct those

**Comment by ajansari95** [comment]
> Due to enforcement of alphabetical order, for pair with cosmoshub and juno, quicksilver should be chain2, not chain 1. Please correct those

sure, fixed it

File: _IBC/quicksilver-secret.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "quicksilver",
+    "client_id": "07-tendermint-28",
+    "connection_id": "connection-14"
+  },
+  "chain_2": {
+    "chain_name": "secret",
```
Comment (JeremyParish69) [inline _IBC/quicksilver-secret.json:9]: oh, sorry, I missed this. 'secret' should be 'secretnetwork' as that's the exact chain_name

Comment (ajansari95) [inline _IBC/quicksilver-secret.json:9]:
```suggestion
    "chain_name": "secretnetwork",
```

Comment (ajansari95) [reply _IBC/quicksilver-secret.json]: no worries, should have looked at other files my bad- fixed it

File: _IBC/quicksilver-secretnetwork.json [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
+      "chain_1": {
+        "channel_id": "channel-52",
+        "port_id": "transfer"
+      },
+      "chain_2": {
+        "channel_id": "channel-65",
+        "port_id": "transfer"
+      },
+      "ordering": "unordered",
+      "version": "ics21",
```
Comment (JeremyParish69) [inline _IBC/quicksilver-secretnetwork.json:24]: what is ICS 21? Is this supposed to be ics20-1?

**Pull request merged**